user-guide/Advanced_Modules/Data_Aggregator/Data_Aggregator_install_setup.md

```diff
@@ -0,0 +1,38 @@
+---
+uid: Data_Aggregator_install_setup
+---
+
+# Data Aggregator installation and setup
+
+The Data Aggregator DxM does not necessarily have to be installed on a server running a DataMiner Agent. You can also install it on a standalone server.
+
+## Prerequisites
+
+- DataMiner 10.2.12 or higher must be installed on the DataMiner Agents you want to use Data Aggregator with.
+
+  > [!NOTE]
+  > If you want to use queries with [ad hoc data sources](xref:Configuring_GQI_feeds#get-ad-hoc-datacustom-data), you will need DataMiner 10.3.3/10.4.0 or higher.
+
+- The CoreGateway DxM must be installed on the DataMiner Agents you want to use Data Aggregator with. To install this DxM, you can use the [DataMiner Cloud Pack](xref:CloudPackages). The DMAs do not have to be connected to dataminer.services.
```

Comment:
If they are connected, they need to be aware of the uninstalled CloudGateway on non internet servers, see the warnings i added in https://github.com/SkylineCommunications/dataminer-docs/pull/1005/files

Reply:
later we will have an offline cloud pack so this issue will be resolved + fix this in the module itself also just in case

Reply:
@JordyGit I think there is some discussion regarding if a separate "offline cloud pack" is really needed. I think the idea is now to rather fix CloudGateway so that it gives no issues anymore when there is no Internet access. Ideally the entire CloudPack gets installed with every DMA install, without having to install it separately afterwards. If the DMA has no Internet access, that should cause issues. (click click click I <3 DataMiner)

Reply:
Indeed, I did not have time to sync with the devs on this yet but we will just let DataMiner install the full Cloud Pack and fix the CloudGateway. We have a task for this on our backlog.